### ISSUE TITLE
@nativescript-rtl/ui is not a valid NativeScript plugin. 

### ISSUE THREAD

**Issue by gimyboya** [nativescript-rtl/ui#1]
I am new to nativescript and this was the first plugin that I tried to install.

getting this error after running `tns plugin add @nativescript-rtl/ui`:
```bash
@nativescript-rtl/ui is not a valid NativeScript plugin. Verify that the plugin package.json file contains a nativescript key and try again.
```

**Comment by xlmnxp** [comment]
I'm working to fix this

**Comment by xlmnxp** [comment]
fixed ce0734306ff83c390f791a4237c46d33c6f8b88e
